exonum/src/sandbox/consensus/round_details.rs

```diff
@@ -1230,8 +1197,7 @@ fn handle_full_propose_wrong_state_hash() {
 
     sandbox.assert_state(Height(1), Round(1));
     //    let tmp_propose = ProposeBuilder::new(&sandbox)
-    //        .with_duration_since_sandbox_time(PROPOSE_TIMEOUT)
-    //        .build();
+    //            //        .build();
```

Comment:
Duplicate `//` here looks weird.

And... do we need this code? Keeping commented out code is not a good practice.